### DISCUSSION TITLE
Change the airfoil geometry

### DISCUSSION THREAD

**Discussion by NM1293** [byuflowlab/FLOWUnsteady#142] [Q&A]
Hello everyone,
I have run the simple wing example available at this link: https://flow.byu.edu/FLOWUnsteady/examples/wing-4p2aoa/

This example creates a simple wing composed by a RAE 101 airfoil section with 12% thickness, as mentioned in the tutorial. If I understand correctly, the information about the airfoil used to generate the wing is passed in the code section 04) MONITORS DEFINITIONS, where the file "xf-rae101-il-1000000.csv" is assigned to the parameter "airfoilpolar" of the function "generate_calc_aerodynamicforce".

I have two questions:

1. Is this the file used to assign the airfoil geometry?;
2. If yes, how is this file structured?;

Thank you

**Comment by EdoAlvarezR** [comment]
Check the file in the [default database](https://github.com/byuflowlab/FLOWUnsteady/tree/master/database) to see the format. This file is used to calculate the parasitic drag (friction + form drag) on the wing, and you can use any other file found in [airfoiltools.com](http://airfoiltools.com/).
